toolkit/src/main/cpp/include/util.h

```diff
@@ -0,0 +1,15 @@
+#include <godot_cpp/templates/local_vector.hpp>
```

Comment:
This needs the copyright statement and header guards - it looks like we've been using `#pragma once` in this repo